### PR TITLE
docs: (IAC-1028) Troubleshooting entry for AWS & GCP non-static kubeconfig files using ansible with orchestration deploy command

### PR DESCRIPTION
**Pull request by jarpat**
### Changes
Adds a DAC troubleshooting entry for AWS & GCP non-static kubeconfig files using ansible with orchestration deploy command

### CODE REVIEW

File: docs/Troubleshooting.md [hunk 1]
```diff
@@ -282,3 +283,63 @@ INGRESS_NGINX_CONFIG:
...
+
+### Solution:
+
+You have a couple of options:
+* If you would still like to deploy the SAS Viya platform with the `sas-orchestration` command with your existing kubernetes configuration file, it is recommended to build Docker image for this project with the [Dockerfile](https://github.com/sassoftware/viya4-deployment/blob/main/Dockerfile) and run it using the [Docker run command](https://github.com/sassoftware/viya4-deployment/blob/main/docs/user/DockerUsage.md).
+* If you created your infrastructure with an Viya 4 IAC projects, you can go back and set `create_static_kubeconfig=true` and run `terraform apply` again to generate a "static" kubeconfig file that is compatible with `sas-orchestration`.
```
Comment (riragh) [inline docs/Troubleshooting.md:342]: Instead of "Viya 4 IAC projects" maybe "viya4-iac-* projects"? or just "Viya4 IAC projects".

Comment (jarpat) [reply docs/Troubleshooting.md:342]: I think switching to viya4-iac-* is a good idea, makes it more clear that it's referring to the IAC project repos. Applying that suggestion

File: docs/Troubleshooting.md [hunk 2]
```diff
@@ -282,3 +283,63 @@ INGRESS_NGINX_CONFIG:
...
+* Using your existing provider based kubernetes configuration and `kubectl` you can alternatively create a new ServiceAccount, associate a service-account-token to it, and grant it admin permissions using RBAC. You should be able to use the ca cert and token from service-account-token to create your own "static" kubernetes configuration file.
+  * See [Kubernetes documentation](https://kubernetes.io/docs/concepts/security/service-accounts/)
+  * Note: this is what the second option above would do for you automatically.
```
[the remark this answers is not in the document]
Comment (jarpat) [reply docs/Troubleshooting.md]: Made the change, you're right, this makes it more clear which option above we are referring to.

File: docs/Troubleshooting.md [hunk 3]
```diff
@@ -282,3 +283,63 @@ INGRESS_NGINX_CONFIG:
...
+
+When running the viya4-deployment project as a Docker container the `sas-orchestration` tooling is run in a slightly different manner to get around this limitation. We make use of `skopeo` to exact the contents of the `sas-orchestration` tooling image directly into our running viya4-deployment container. Since in our Dockerfile we include the installation of the required authentications binaries for GCP and AWS, the `sas-orchestration` tooling is able to make use of them and successfully connect to the kubernetes cluster.
+
+### Solution:
+
+You have a couple of options:
+* If you would still like to deploy the SAS Viya platform with the `sas-orchestration` command with your existing kubernetes configuration file, it is recommended to build Docker image for this project with the [Dockerfile](https://github.com/sassoftware/viya4-deployment/blob/main/Dockerfile) and run it using the [Docker run command](https://github.com/sassoftware/viya4-deployment/blob/main/docs/user/DockerUsage.md).
```
[the remark this answers is not in the document]
Comment (jarpat) [reply docs/Troubleshooting.md]: Change made

**Pull request merged**